app/src/main/java/org/americanairlines/our1grouprestaurantmapproject/model/NearbyPlacesModel.kt

```diff
@@ -0,0 +1,20 @@
+package org.americanairlines.our1grouprestaurantmapproject.model
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.google.android.gms.maps.model.LatLng
+
+/*
+ * This is a place to put data our app will actually use (as well as providing a simple caching object.)
+ * this class should be modified to reflect the data we want.
+ * The primary key is nullable since, well, we cannot guarentee
+ */
+
+@Entity
+data class NearbyPlacesModel(
+        @PrimaryKey(autoGenerate = true) val id : Int?,
```

Comment:
Primary key should not be nullable

Reply:
If room complains, then I'll change it, but only room should assign ids.